spyder/plugins/ipythonconsole/comms/tests/test_comms.py

```diff
@@ -121,7 +121,7 @@ def is_alive():
 
     # Bypass the target system as this is not what is being tested
     frontend_comm._comm_open(
```

Comment:
```suggestion
    frontend_comm._comm_open(commB, {'content': {}})
```

This fits in a single line now.